src/interface/meshblock_data.cpp

```diff
@@ -79,9 +77,7 @@ void MeshBlockData<T>::Add(const std::string &label, const Metadata &metadata) {
   } else {
     auto sv = std::make_shared<CellVariable<T>>(label, arrDims, metadata);
     Add(sv);
-    if (metadata.IsSet(Metadata::FillGhost)) {
-      sv->allocateComms(pmy_block);
-    }
+    sv->allocateComms(pmy_block);
```

Comment:
Same as above regarding why no FillGhost check is required any more.

Reply:
See above